src/handle_event.py

```diff
@@ -0,0 +1,14 @@
+from .github_sdk import send_trace
+
+
+def handle_event(data, headers):
```

Comment:
Moving this code helps with testing since we don't initiate the flask app.

Reply:
we could probably move to a `create_app` pattern which avoids import-time side-effects, though probably best for another PR!